internal/clients/http/sources/sources_client.go

```diff
@@ -77,6 +77,9 @@ func (c *sourcesClient) Ready(ctx context.Context) error {
 	if resp == nil {
 		return fmt.Errorf("ready call: empty response: %w", clients.ErrUnexpectedBackendResponse)
 	}
+	if resp.JSON400 != nil {
+		logger.Warn().Bytes("body", resp.Body).Int("status", resp.StatusCode()).Msg("Get authentication from sources returned 4xx")
```

Comment:
I've failed to actually send this comment for some reason:

Why are we logging it in separate field and not the message? :thinking: